src/Chapter2.hs

```diff
@@ -534,6 +552,7 @@ For example, we can patch the previous function to count the number of
 steps we need to take in order to reduce the number to zero.
 
 🤔 Blitz question: can you guess what this number represents?
+stack size?
```

Comment:
Yes, but what does this stack size represent? :upside_down_face:

Reply:
:neckbeard: hehe, I gave that answer, because I wasn't really following the question at the time. I read over it again and I realized that you meant in this specific example function what did the acc represent, yeah? 

I updated my answer to "smallest power of 2 greater than or equal to n?", but as you can see with the question mark at the end of it, lol, I'm still a bit unsure if that is what you are looking for =))

Either way, thanks so much for all these comments!

Reply:
This is exactly what we are looking for :+1: Absolutely correct answer :clap: